user-journey-tests/helpers/testHelpers/checkAnswers.js

```diff
@@ -87,3 +89,25 @@ export const validateAndAdjustEmail = async (
 
   await validateElementVisibleAndText(valueElement, inputEmail)
 }
+
+export const validateOnFarmErrorHandling = async (
+  changeElement,
+  checkAnswersHeading,
+  final = true
+) => {
+  await selectElement(changeElement)
+  await toFromFarmPage.selectOnFarmAndContinue()
+  await exitPage.verifyPageHeadingAndTitle(exitPage.pageHeading)
```

Comment:
does this function not know its own title?

Reply:
Nah you have to sub it in, that's just calling the method from page.js

Reply:
@eoin-corr-git  - we could do a bit of a refactor here. 

It used to be that each page had a different field for the heading & title, but now we've normalised it, you can call `this.pageTitle` or `this.pageHeading` in the page base class.

Reply:
But I think that's independent of this PR, because that code is everywhere ;)